web/source/kmwbase.js

```diff
@@ -1084,14 +1084,18 @@ if(!window['keyman']['loaded']) {
      * @param       {Event}      e        event
      * @return      {Object}              HTML element
      */       
-    util.eventTarget = function(e)
-    {
-      if(e && e.target)        // most browsers
+    util.eventTarget = function(e) {
```

Comment:
I was surprised to find that this already existed, but it wasn't integrated well.

Additionally - there is significant evidence suggesting that `keymanweb._GetEventObject(e);` should be called within this function instead of outside it, despite the inconsistency of this occurring in our present code.  The 'extra' block (about `window.event.srcElement`) is actually addressed by that function, along with a few extra permutations.